src/routes/committee/setlist/+page.svelte

```diff
@@ -50,10 +54,19 @@
 	let instrument: string
 
 	async function RemoveSongFromSetlist() {
+		// Remove the song from the setlist
 		const editionRef = doc(db, editionId)
 		updateDoc(editionRef, {
 			songs: arrayRemove(doc(db, 'songs', data.songs[selectedSong].id))
 		})
+
+		// If the song was not a suggestion (but added directly to the setlist), delete it from the database completely
+		const suggestionIds = (await getSuggestedSongs()).map((song) => song.id)
+		if (!suggestionIds.includes(data.songs[selectedSong].id)) {
+			const suggestionDoc = doc(db, 'songs', data.songs[selectedSong].id) 
+			await deleteDoc(suggestionDoc)
+		}
```

Comment:
De getSuggestedSongs functie heeft al een ids filter, als je daar de benodigde songid aan meegeeft hoef je daarna alleen maar te checken of de lijst leeg is